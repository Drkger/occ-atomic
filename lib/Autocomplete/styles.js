import spacing from '../subatomic/spacing';

export default {
    autoComplete: {
        position: 'relative'
    },
    droplist: {
        position: 'absolute',
        zIndex: 2,
        top: `calc(100% - ${spacing.xTiny}px)`,
        left: 0,
        width: '100%'
    }
};
