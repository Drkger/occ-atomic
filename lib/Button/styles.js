import colors from '../subatomic/colors';
import fonts from '../subatomic/fonts';
import icons from '../subatomic/icons';

export default {
    btn: {
        alignItems:'flex-start',
        boxSizing:'border-box',
        display:'inline-block',
        marginBottom:'0',
        textAlign:'center',
        whiteSpace:'nowrap',
        verticalAlign:'middle',
        touchAction:'manipulation',
        cursor:'pointer',
        userSelect:'none',
        padding:'0 30px',
        height:'30px',
        lineHeight:'24px',
        fontFamily:fonts.body,
        fontSize:'14px',
        fontWeight:'600',
        position:'relative',
        transition:'0.3s all',
        maxWidth:'100%',
        borderRadius:'5px',
        textDecoration:'none',
        '&:focus': {
            outline:'none'
        },
        '&:hover': {
            textDecoration:'none'
        }
    },
    cont: {
        display:'flex',
        width:'100%',
        height:'100%',
        alignItems:'center',
        justifyContent:'center',
        pointerEvents:'none'
    },
    loadCont: {
        display:'flex',
        width:'100%',
        height:'100%',
        alignItems:'center',
        justifyContent:'center',
        position:'absolute',
        top:'0',
        left:'0',
        pointerEvents:'none'
    },
    // Themes
    main: {
        background:colors.pink,
        color:colors.white,
        border:`1px solid ${colors.pink}`,
        '&:hover': {
            background:colors.pinkH,
            color:colors.white,
            borderColor:colors.pinkH
        },
        '&:active': {
            background:colors.pinkA,
            color:colors.white,
            borderColor:colors.pinkA
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&$loading': {
            background:colors.pinkA,
            borderColor:colors.pinkA
        },
        '&$disabled': {
            background:`${colors.grey1} !important`,
            color:`${colors.white} !important`,
            border:`1px solid ${colors.grey1} !important`
        }
    },
    secondary: {
        background:'transparent',
        color:colors.pink,
        border:`1px solid ${colors.pink}`,
        '&:hover': {
            background:colors.pinkH,
            color:colors.white,
            borderColor:colors.pinkH,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '&:active': {
            background:colors.pinkA,
            color:colors.white,
            borderColor:colors.pinkA,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.pink])) : ''
        },
        '&$loading': {
            background:colors.pinkA,
            borderColor:colors.pinkA
        },
        '&$disabled': {
            background:`${colors.grey1} !important`,
            color:`${colors.white} !important`,
            border:`1px solid ${colors.grey1} !important`
        }
    },
    white: {
        background:'transparent',
        color:colors.white,
        border:`1px solid ${colors.white}`,
        '&:hover': {
            background:'rgba(255,255,255,0.2)',
            color:colors.white,
            borderColor:colors.white,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '&:active': {
            background:'rgba(255,255,255,0.2)',
            color:colors.white,
            borderColor:colors.white,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
            }
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&$loading': {
            background:'rgba(255,255,255,0.2)',
            borderColor:colors.white
        },
        '&$disabled': {
            background:`transparent !important`,
            color:`rgba(255,255,255,0.6) !important`,
            border:`1px solid rgba(255,255,255,0.6) !important`,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon(['rgba(255,255,255,0.6)'])) : ''
            }
        }
    },
    flat: {
        background:'transparent',
        color:colors.pink,
        border:`1px solid transparent`,
        '&:hover': {
            background:colors.grey3,
            color:colors.pink,
            borderColor:colors.grey3,
        },
        '&:active': {
            background:colors.grey5,
            color:colors.pink,
            borderColor:colors.grey5,
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.pink])) : ''
        },
        '& $loadIcon': {
            background: icons.base(icons.loading.icon([colors.pink]))
        },
        '&$loading': {
            background:colors.white,
            borderColor:colors.white
        },
        '&$disabled': {
            background:'transparent !important',
            color:`${colors.grey1} !important`,
            border:`1px solid transparent !important`,
            '& i': {
                background: props => props.icon ? icons.base(icons[props.icon].icon([colors.grey1])) : ''
            },
            '& $loadIcon': {
                background: icons.base(icons.loading.icon([colors.grey1]))
            },
        }
    },
    blue: {
        background:colors.blue,
        color:colors.white,
        border:`1px solid ${colors.blue}`,
        '&:hover': {
            background:colors.blueH,
            color:colors.white,
            borderColor:colors.blueH,
        },
        '&:active': {
            background:colors.blueA,
            color:colors.white,
            borderColor:colors.blueA,
        },
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&$loading': {
            background:colors.blueA,
            borderColor:colors.blueA
        },
        '&$disabled': {
            background:`${colors.grey1} !important`,
            color:`${colors.white} !important`,
            border:`1px solid ${colors.grey1} !important`
        }
    },
    loading: {
        pointerEvents:'none',
        '& $cont': {
            visibility:'hidden'
        }
    },
    disabled: {
        background:colors.grey1,
        color:colors.white,
        border:`1px solid ${colors.grey1}`,
        cursor:'default',
        pointerEvents:'none',
        '& i': {
            background: props => props.icon ? icons.base(icons[props.icon].icon([colors.white])) : ''
        },
        '&:hover, &:active, &:focus': {
            background:colors.grey1,
            color:colors.white,
            border:`1px solid ${colors.grey1}`
        }
    },
    // Sizes
    md: {
        fontSize:'16px',
        height:'40px',
        lineHeight:'28px',
        '& i': {
            width:'28px',
            height:'28px'
        }
    },
    lg: {
        fontSize:'18px',
        height:'46px',
        lineHeight:'32px',
        '& i': {
            width:'32px',
            height:'32px'
        }
    },
    // Block
    block: {
        display:'block',
        width:'100%'
    },
    // Icon
    icon: {
        width:'24px',
        height:'24px',
        display:'inline-block',
        transition:'0.3s all'
    },
    '@keyframes rotate': {
        from: {transform: 'rotate(0deg)'},
        to: {transform: 'rotate(360deg)'}
    },
    loadIcon: {
        width:'24px',
        height:'24px',
        display:'inline-block',
        transition:'0.3s all',
        animation:'rotate 1s infinite linear',
        background: icons.base(icons.loading.icon())
    }
};