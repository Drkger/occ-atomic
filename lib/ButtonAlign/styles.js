export default {
    sm: {
        marginTop:'19px'
    },
    md: {
        marginTop:'9px'
    },
    lg: {
        marginTop:'3px'
    }
};