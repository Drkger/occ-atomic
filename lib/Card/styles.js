import colors from '../subatomic/colors';
import fonts from '../subatomic/fonts';
import shadows from '../subatomic/shadows';

const styles = {
    card: {
        background:colors.white,
        borderRadius:'5px',
        padding:'30px 20px'
    },
    cardNoPadding: {
        extend: 'card',
        padding:'0'
    }
};

const pushShadows = function() {
    for (let i = 0; i<7; i++) {
        styles[`shadow${i}`] = {
            boxShadow: shadows[`lvl${i}`]
        };
    }
};

pushShadows();

export default styles;