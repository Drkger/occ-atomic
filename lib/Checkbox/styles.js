import colors from '../subatomic/colors';
import spacing from '../subatomic/spacing';
import icons from '../subatomic/icons';

export default {
    cont: {
        paddingTop:spacing.tiny,
        paddingBottom:spacing.tiny,
        boxSizing:'border-box',
        display:'flex',
        alignItems:'center',
        cursor:'pointer',
        outline:'0',
        '&:after': {
            content:'""',
            display:'table',
            clear:'both'
        },
        '&:hover $check:after': {
            background:icons.base(icons.check.icon([colors.grey200]))
        }
    },
    check: {
        width:spacing.base,
        height:spacing.base,
        position:'relative',
        '&:before': {
            content:'""',
            width:spacing.small,
            height:spacing.small,
            borderRadius:4,
            position:'absolute',
            top:'50%',
            left:'50%',
            transform:'translate(-50%, -50%)',
            border:`1px solid ${colors.grey200}`,
            background:colors.bgWhite
        },
        '&:after': {
            content:'""',
            width:18,
            height:18,
            marginTop:-1,
            position:'absolute',
            top:'50%',
            left:'50%',
            transform:'translate(-50%, -50%)',
            transition:'0.3s all',
            background:'transparent'
        }
    },
    active: {
        '& $check:after': {
            background:[icons.base(icons.check.icon([colors.prim])), '!important']
        }
    },
    disabled: {
        opacity:0.4,
        pointerEvents:'none'
    },
    label: {
        marginLeft:spacing.tiny,
        cursor:'pointer'
    }
};
