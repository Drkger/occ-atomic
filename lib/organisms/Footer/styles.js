import colors from '../../subatomic/colors';
import fonts from '../../subatomic/fonts';
import grid from '../../subatomic/grid';
import icons from '../../subatomic/icons';

export default {
    footer: {
        backgroundColor:colors.black,
        height:'25px',
        margin:'0',
        padding:'0 40px',
        fontSize:'12px',
        color:colors.grey2,
        fontFamily:fonts.title
    },
    footerList: {
        listStyle:'none',
        width:'auto',
        margin:'0',
        padding:'4px 0 0 0'
    },
    listElement: {
        listStyle:'none',
        float:'left',
        margin:'0 50px 0 0',
        padding:'0',
        position:'relative'
    },
    link: {
        color:colors.grey2,
        cursor:'pointer',
        textDecoration:'none',
        '&:hover, &:active, &:focus': {
            color:colors.white,
            textDecoration:'none'
        }
    },
    [`@media (max-width: ${grid.md-1}px)`]: {
        footer: {
            display: 'none !important'
        }
    },
    dropdownMenu: {
        display:'none',
        background:colors.black,
        position:'absolute',
        bottom:'calc(100% + 10px)',
        left:'50%',
        transform:'translateX(-50%)',
        minWidth:'160px',
        padding:'5px 0',
        fontSize:'12px',
        textAlign:'left',
        listStyle:'none',
        borderRadius:'4px',
        boxShadow:'0 6px 12px rgba(0,0,0,.175)',
        zIndex:'2'
    },
    dropdownElement: {
        float:'left',
        listStyle:'none'
    },
    dropdownLink: {
        color:colors.grey2,
        textDecoration:'none',
        padding:'3px 18px',
        lineHeight:'1.42857143',
        display:'block',
        '&:hover': {
            color:colors.white,
            textDecoration:'none'
        }
    }
};