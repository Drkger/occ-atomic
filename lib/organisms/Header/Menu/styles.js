import colors from '../../../subatomic/colors';
import grid from '../../../subatomic/grid';
import icons from '../../../subatomic/icons';

export default {
    menu: {
        marginTop:'5px',
        display:'inline-block',
        cursor:'pointer !important',
    },
    btnContainer: {
        position:'relative',
        display:'inline-block',
        verticalAlign:'middle'
    },
    btnMenu: {
        cursor:'pointer',
        background:'none',
        border:'none',
        padding:'5px 10px 2px 10px',
        marginLeft:'10px',
        color:colors.white,
        transition:'0.3s all',
        outline:'none !important',
        borderRadius:'3px',
        '&:hover': {
            color:colors.blue,
            background:colors.white
        },
        '&:hover $menuIcon': {
            background:icons.base(icons.menu.icon(["#1476fb"]))
        }
    },
    btnMenuOpen: {
        color:colors.blue,
        background:colors.white,
        '& $menuIcon': {
            background:icons.base(icons.menu.icon(["#1476fb"]))
        }
    },
    userName: {
        paddingRight:'5px',
        margin:'0',
        display:'inline-block',
        overflow:'hidden',
        textOverflow:'ellipsis',
        maxWidth:'65px',
        whiteSpace:'nowrap'
    },
    menuIcon: {
        width:icons.menu.width,
        height:icons.menu.height,
        display:icons.menu.display,
        transition:'0.3s all',
        background:icons.base(icons.menu.icon(["#FFFFFF"]))
    },
    dropdownMenu: {
        position:'absolute',
        top:'100%',
        zIndex:'1000',
        display:'none',
        float:'left',
        minWidth:'160px',
        padding:'5px 0',
        fontSize:'14px',
        textAlign:'left',
        listStyle:'none',
        backgroundColor:colors.white,
        backgroundClip:'padding-box',
        border:'1px solid rgba(0,0,0,.15)',
        boxShadow:'0 6px 12px rgba(0,0,0,.175)',
        margin:'13px 0 0 -187px',
        right:'0'
    },
    dropdownMenuOpen: {
        display:'block'
    },
    listAnchor: {
        textDecoration:'none',
        display:'block',
        padding:'0 10px',
        margin:'5px 0 5px 0',
        clear:'both',
        fontWeight:'400',
        lineHeight:'1.42857143',
        whiteSpace:'nowrap',
        color:'#000',
        cursor:'pointer',
        '&:hover, &:focus': {
            textDecoration:'none',
            color:colors.blue
        }
    },
    signOut: {
        color:`${colors.pink} !important`,
        margin:'10px 0 5px 0'
    },
    separator: {
        height: '1px',
        background:'#ebebeb'
    },
    [`@media (max-width: ${grid.sm-1}px)`]: {
        dropdownMenu: {
            width:'200px'
        }
    }
};