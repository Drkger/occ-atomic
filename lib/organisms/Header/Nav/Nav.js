import React from 'react';
import PropTypes from 'prop-types';
import injectSheet from 'react-jss';

class Nav extends React.Component {
    render() {
        const { classes, active, navigate } = this.props;
        return (
            <ul className={classes.menu}>
                <li onClick={navigate} className={`${classes.listItem} ${active == "actividad" ? classes.activeItem : ''}`} id="actividad">
                    <i className={`${classes.dashboard} ${classes.icon}`} />
                    <a className={classes.listAnchor}>Mi actividad</a>
                </li>
                <li onClick={navigate} className={`${classes.listItem} ${active == "publicaciones" ? classes.activeItem : ''}`} id="publicaciones">
                    <i className={`${classes.publications} ${classes.icon}`} />
                    <a className={classes.listAnchor}>Publicaciones</a>
                </li>
                <li onClick={navigate} className={`${classes.listItem} ${active == "talento" ? classes.activeItem : ''}`} id="talento">
                    <i className={`${classes.talent} ${classes.icon}`} />
                    <a className={classes.listAnchor}>Talento</a>
                </li>
            </ul>
        );
    }
}

Nav.propTypes = {
    classes: PropTypes.object,
    active: PropTypes.string,
    navigate: PropTypes.func
};

export default Nav;