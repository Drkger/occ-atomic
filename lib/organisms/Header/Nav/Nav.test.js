import React from 'react';
import { shallow } from 'enzyme';
import Nav from './Nav';
import styles from './styles';
import NavJSS from './index';

const reduceClasses = (prev, curr) => Object.assign({}, prev, { [curr]: curr });

const classes = Object.keys(styles).reduce(reduceClasses, {});

const buttons = [
    { id: 'item1', icon:'dashboard', label:'Item 1' },
    { id: 'item2', icon:'publications', label:'Item 2' },
    { id: 'item3', icon:'talent', label:'Item 3' }
]

describe("Nav", () => {

    it('matches the snapshot', () => {
        const wrapper = shallow(
            <Nav classes={classes} buttons={buttons} />
        );
        expect(wrapper).toMatchSnapshot();
    });

    it('renders all the buttons', () => {
        const wrapper = shallow(
            <Nav classes={classes} buttons={buttons} />
        );
        expect(wrapper.find('li').length).toBe(3);
    });

    it('calls the onClick function', () => {
        const onClick = jest.fn();
        const wrapper = shallow(
            <Nav classes={classes} navigate={onClick} buttons={buttons} />
        );
        wrapper.find('li').at(1).simulate('click');
        expect(onClick.mock.calls.length).toBe(1);
    });

    it('sets an active button', () => {
        const wrapper = shallow(
            <Nav classes={classes} active="item1" buttons={buttons} />
        );
        expect(wrapper.find('li').at(0).hasClass('activeItem')).toBe(true);
    });

});

describe("NavJSS", () => {

    it('matches the snapshot', () => {
        const wrapper = shallow(
            <NavJSS buttons={buttons} />
        );
        expect(wrapper).toMatchSnapshot();
    });

});

describe("Nav styles", () => {

    it('matches the snapshot', () => {
        expect(styles).toMatchSnapshot();
    });

});