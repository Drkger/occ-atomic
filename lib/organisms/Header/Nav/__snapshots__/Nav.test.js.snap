// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nav
    buttons={
      Array [
        Object {
          "icon": "dashboard",
          "id": "item1",
          "label": "Item 1",
        },
      ]
    }
    classes={
      Object {
        "@media (max-width: 767px)": "@media (max-width: 767px)",
        "@media (min-width: 768px)": "@media (min-width: 768px)",
        "activeItem": "activeItem",
        "icon": "icon",
        "listAnchor": "listAnchor",
        "listItem": "listItem",
        "menu": "menu",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <li
          className="listItem "
          id="item1"
          onClick={undefined}
        >
          <Jss(Icon)
            className="icon"
            colors={undefined}
            iconName="dashboard"
          />
          <a
            className="listAnchor"
          >
            Item 1
          </a>
        </li>,
      ],
      "className": "menu",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "item1",
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Jss(Icon)
              className="icon"
              colors={undefined}
              iconName="dashboard"
            />,
            <a
              className="listAnchor"
            >
              Item 1
            </a>,
          ],
          "className": "listItem ",
          "id": "item1",
          "onClick": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "icon",
              "colors": undefined,
              "iconName": "dashboard",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Item 1",
              "className": "listAnchor",
            },
            "ref": null,
            "rendered": "Item 1",
            "type": "a",
          },
        ],
        "type": "li",
      },
    ],
    "type": "ul",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li
            className="listItem "
            id="item1"
            onClick={undefined}
          >
            <Jss(Icon)
              className="icon"
              colors={undefined}
              iconName="dashboard"
            />
            <a
              className="listAnchor"
            >
              Item 1
            </a>
          </li>,
        ],
        "className": "menu",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "item1",
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Jss(Icon)
                className="icon"
                colors={undefined}
                iconName="dashboard"
              />,
              <a
                className="listAnchor"
              >
                Item 1
              </a>,
            ],
            "className": "listItem ",
            "id": "item1",
            "onClick": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "icon",
                "colors": undefined,
                "iconName": "dashboard",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Item 1",
                "className": "listAnchor",
              },
              "ref": null,
              "rendered": "Item 1",
              "type": "a",
            },
          ],
          "type": "li",
        },
      ],
      "type": "ul",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Nav styles matches the snapshot 1`] = `
Object {
  "@media (max-width: 767px)": Object {
    "activeItem": Object {
      "& $icon": Object {
        "marginBottom": "14px",
      },
      "& $listAnchor": Object {
        "display": "none",
      },
      "&:after": Object {
        "background": "#06f7fa",
        "borderRadius": "50%",
        "bottom": "0",
        "content": "\\"\\"",
        "display": "block",
        "height": "4px",
        "left": "50%",
        "marginBottom": "8px",
        "position": "absolute",
        "transform": "translateX(-50%)",
        "width": "4px",
      },
    },
    "icon": Object {},
    "listAnchor": Object {},
    "listItem": Object {
      "&:last-child": Object {
        "marginRight": "0",
      },
      "margin": "0 25px 0 0",
    },
    "menu": Object {
      "clear": "both",
      "margin": "0",
      "textAlign": "center",
    },
  },
  "@media (min-width: 768px)": Object {
    "listItem": Object {
      "float": "left",
    },
    "menu": Object {
      "float": "left",
    },
  },
  "activeItem": Object {
    "& $icon, &:hover $icon": Object {
      "filter": "brightness(1)",
    },
    "& $listAnchor, &:hover $listAnchor": Object {
      "color": "#ffffff",
    },
  },
  "icon": Object {},
  "listAnchor": Object {
    "color": "#104791",
    "display": "block",
    "lineHeight": "20px",
    "marginBottom": "2px",
    "marginTop": "-4px",
    "padding": "0",
    "position": "relative",
    "textAlign": "center",
    "textDecoration": "none !important",
    "transition": "0.3s all",
  },
  "listItem": Object {
    "&:hover $icon": Object {
      "filter": "brightness(0.8)",
    },
    "&:hover $listAnchor": Object {
      "color": "#224099",
    },
    "cursor": "pointer",
    "display": "inline-block",
    "fontSize": "12px",
    "margin": "7px 30px 0 0",
    "padding": "5px 0 0 0",
    "position": "relative",
    "textAlign": "center",
    "width": "auto",
  },
  "menu": Object {
    "background": "#1476fb",
    "listStyle": "none",
    "margin": "0 10px",
    "paddingLeft": "0",
    "textAlign": "center",
  },
}
`;

exports[`NavJSS matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Jss(Nav)
    buttons={
      Array [
        Object {
          "icon": "dashboard",
          "id": "item1",
          "label": "Item 1",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "buttons": Array [
        Object {
          "icon": "dashboard",
          "id": "item1",
          "label": "Item 1",
        },
      ],
      "classes": Object {
        "activeItem": "Nav-activeItem-0-1-4",
        "icon": "Nav-icon-0-1-3",
        "listAnchor": "Nav-listAnchor-0-1-5",
        "listItem": "Nav-listItem-0-1-2",
        "menu": "Nav-menu-0-1-1",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "buttons": Array [
          Object {
            "icon": "dashboard",
            "id": "item1",
            "label": "Item 1",
          },
        ],
        "classes": Object {
          "activeItem": "Nav-activeItem-0-1-4",
          "icon": "Nav-icon-0-1-3",
          "listAnchor": "Nav-listAnchor-0-1-5",
          "listItem": "Nav-listItem-0-1-2",
          "menu": "Nav-menu-0-1-1",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
